storagenode/contact/chore.go

```diff
@@ -11,30 +11,19 @@ import (
 	"github.com/zeebo/errs"
 	"go.uber.org/zap"
 	"golang.org/x/sync/errgroup"
-	"gopkg.in/spacemonkeygo/monkit.v2"
 
 	"storj.io/storj/internal/sync2"
-	"storj.io/storj/pkg/kademlia"
 	"storj.io/storj/pkg/pb"
 	"storj.io/storj/pkg/transport"
 	"storj.io/storj/storagenode/trust"
 )
 
-var mon = monkit.Package()
-
-// Config contains configurable parameters for contact chore
-type Config struct {
-	Interval time.Duration `help:"how frequently the node contact chore should run" releaseDefault:"1h" devDefault:"30s"`
-	// MaxSleep should remain at default value to decrease traffic congestion to satellite
-	MaxSleep time.Duration `help:"maximum duration to wait before pinging satellites" releaseDefault:"45m" devDefault:"0s" hidden:"true"`
-}
-
 // Chore is the contact chore for nodes announcing themselves to their trusted satellites
 //
 // architecture: Chore
```

Comment:
whats up with this comment?

Reply:
I'm not sure... @cam-a do you know?

Reply:
looks to be for graphing https://github.com/storj/storj/commit/a801fab66a49ed37afd7416b64eb2e058c826fef

Reply:
oh interesting